src/components/pages/Project.vue

```diff
@@ -100,6 +100,9 @@
                       Meilensteine
                     </v-card-text>
                   </v-system-bar>
+                  <v-card-text v-if="project.milestones.length == 0">
```

Comment:
=== statt ==
https://www.jstips.co/en/javascript/use_===_instead_of_==/#:~:text=)%20operator%20performs%20an%20automatic%20type,faster%20(jsPref)%20than%20%3D%3D%20.

Reply:
Vereinfacht gesagt, gewöhn dir in Javascript ruhig an immer === zu benutzen, außer wenn du auf null oder undefined prüfst.

Reply:
Danke für den Hinweis 👍